### ISSUE TITLE
OSHI-5.1.1: LinuxOSProcess start time wrong

### ISSUE THREAD

**Issue by motasus** [oshi/oshi#1255]
It looks like forgotten parentheses in assignment computation [here](../blob/oshi-parent-5.1.1/oshi-core/src/main/java/oshi/software/os/linux/LinuxOSProcess.java#L284):
```java
this.startTime = LinuxOperatingSystem.BOOTTIME
                + statArray[ProcPidStat.START_TIME.ordinal()] * 1000L / LinuxOperatingSystem.getHz();
```
Mixing seconds (BOOTTIME) and milliseconds. This also leads to invalid uptime.

**Comment by dbwiddis** [comment]
That's been in there an incredibly long time.... surprised nobody found it before now!

**Comment by dbwiddis** [comment]
It's not quite so simple as parentheses as the final multiplication by 1000 would lose precision.  Could cast to double and back but possibly better to multiply boottime by hz and then group the first two terms in same units.

**Comment by dbwiddis** [comment]
@motasus can you take a look at the fix in #1256 and see if you think it's correct?

**Comment by motasus** [comment]
Yes, this looks better and SystemInfoTest now prints reasonable process CPU load.
